ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_extrema_finding.py

```diff
@@ -344,3 +344,45 @@ def test_numpy_fmax(
         out=None,
         where=where,
     )
+
+
+# fmin
+@handle_frontend_test(
+    fn_tree="numpy.fmin",
+    dtype_and_inputs=helpers.dtype_and_values(
```

Comment:
Hey here we will use `np_frontend_helpers.dtypes_values_casting_dtype` to generate the casting dtypes as well